docs/source/pages/lightning.rst

```diff
@@ -150,4 +185,24 @@ If ``on_epoch`` is True, the logger automatically logs the end of epoch metric v
                 self.metric(outputs['preds'], outputs['target'])
                 self.log('metric', self.metric)
 
+.. note::
+
+    When using any Modular metric, calling ``self.metric(...)`` or ``self.metric.forward(...)`` serves the dual purpose of calling ``self.metric.update()``
+    on its input and simultaneously returning the value of the metric over the provided input. So if you are logging a metric on epoch-level, it
+    is recommended to call ``self.metric.update()`` directly, to avoid the extra computation.
```

Comment:
```suggestion
    When using any Modular metric, calling ``self.metric(...)`` or ``self.metric.forward(...)`` serves the dual purpose of calling ``self.metric.update()``
    on its input and simultaneously returning the metric value over the provided input. So if you are logging a metric on epoch-level, it
    is recommended to call ``self.metric.update()`` directly to avoid the extra computation.
```